exercises/concept/amusement-park-improvements/.docs/instructions.md

```diff
@@ -13,7 +13,7 @@ Attendee.new(100).has_pass?
 
 ## 2. Check if an attendee fits a ride
 
-Implement the `Attendee#fits_ride?` method to see if an attendee fits a ride based on their height.
+Implement the `Attendee#fits_ride?` method to see if an attendee fits a ride based on their height. The ride's required minimum height is provided as an argument. An attendee must have height greater than or equal to ride's required minimum height.
```

Comment:
This seems about right.

One sentence per line, though:

```suggestion
Implement the `Attendee#fits_ride?` method to see if an attendee fits a ride based on their height.
The ride's required minimum height is provided as an argument.
 An attendee must have height greater than or equal to ride's required minimum height.
```

Reply:
The name being in the pre-written part of the solution file is probably
the intended source of truth, rather than the instruction file.

After you opened up the code where you were intended to write, was it
then apparent?

Reply:
After I read the instruction I was confused about what I had to do. When I looked at the source code, I read the name of the argument and it became clear what I had to do. So yes. 

When I saw the instructions of methods in some of the exercises and it made sense to include these instructions. 

I will do the above-proposed changes.

Reply:
So the remaining question, from a mentor to a student, does it make sense to have the explanation in the README, or have the truth in scaffolding that is given, since "Continuing your work with the amusement park, you are tasked with writing some utility methods to facilitate checking if an attendee can use a ride" and that implies a task of becoming familiar, or presuming that you are already familiar with this code.

Since this comes after the exercise that you have done to complete the _Amusement Park_ exercise, the code that has this is already implemented by you, and arguably, is something that you already know.

I am trying to find out if there is value in having a student make that connection, as I do think you have demonstrated your self that it is not a leap, but a small step to have the question, and then to immediately find it in a place that you have in fact already done, as you have completed the prior exercise, and while we might have chosen a different name in the file that already exists, it probably is very close to your implementation.  The problem solved there has not changed.

Reply:
As a student learning, I think the clear instructions makes more sense. But it's also true that after seeing the method declaration and arguments,  I understood the assignment.

That's also because I am not new to programming. I could guess what needs to be done and I tried that and the test passed. But doesn't that make it hit and try solution? Having clear instruction in the readme, will allow student to think before even going to the implementation class and maybe think of a way to implement it. 

What do you think?

Reply:
This is the difference between concept exercises and practice exercises.  The concept exercises are more geared for "Learning to Program" and the practice exercises are geared for exactly what you stated " that is because I am not new to programming".

The tests for practice exercises are not really meant to be hidden from the student, they are a form of communication and should be used by the student for clarification. 

So you may be right for this as a concept exercise, to maybe write it out.  It would be even better if the students solution for their prior exercise could literally be used for what they (and we) would have for the base of that which is to be improved.

That reminds me, perhaps the pull request title should be updated to indicate the exercise it addresses:

"Amusement Park Improvement: Update instruction for `fits_ride?` method."

Reply:
@LeonardEuler Thanks for suggesting the improved title for the PR.  I can find it easier in my list of PR's.  And of course thank you @praveendhawan for updating the title.

I will go with whatever you think should be done for the sake of the concept exercise.  I realize that the student may not see the test file if they are using the web interface.